### PR TITLE
Feat: Improvements

### PR DESCRIPTION
**Pull request by Rohitprasad83**
- Changed the weather api from city to latitude and longitude
- added API key using .env file
- added Links component

- Ignore .env file, will remove it once this PR is complete.
---
I9PhC84hpG

### CODE REVIEW

File: src/components/Links.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import { v4 as uuidv4 } from 'uuid'
+
+export function Links() {
+  const [openLinks, setOpenLinks] = useState(false)
+  const [createLink, setCreateLink] = useState(false)
+  const [name, setName] = useState('')
+  const [link, setLink] = useState('')
+  const [links, setLinks] = useState([])
+
+  function createLinkHandler() {
```
Comment (arpit01923) [inline src/components/Links.jsx:11]: use ES5 and later use ES6, maintain consistency

**Review by arpit01923 — APPROVED** [review]
- clean code
- good folder structure
- one suggestion, maintain consistency

**Review by aniketsingh13 — APPROVED** [review]
Ui looks good
code structure look good
code is easy to read
good to merge

File: src/components/Links.jsx (hunk 1 above)
Comment (Rohitprasad83) [reply src/components/Links.jsx:11]: yes will do it

**Pull request merged**